packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -100,6 +100,30 @@ function normalizeInstagramTVURL(url: IUrl) {
   };
 }
 
+// Instagram
+// - www.instagram.com/p/:id
+// - www.instagr.am/p/:id
+// - instagram.com/p/:id
+// - instagr.am/p/:id
```

Comment:
I think these examples in the comments need to be updated - these aren't reel URLs